Started on new website

pwnate opened the pull request:
Updated the bootstrap theme. Changed main page layout and added a small
amount of content.

Commit:
Started on new website

Updated the bootstrap theme. Changed main page layout and added a small
amount of content.

saibotsivad commented:
I'm on my phone so I don't know if you merged this already, but I think if you merge it it'll get auto deployed? Feel free to merge as much as you want!

On January 22, 2016 7:54:15 PM CST, Nathan Davis notifications@github.com wrote:

> Updated the bootstrap theme. Changed main page layout and added a small
> amount of content.
> You can view, comment on, or merge this pull request online at:
> 
>  https://github.com/saibotsivad/omaha3dprint.com/pull/2
> 
> -- Commit Summary --
> - Started on new website
> 
> -- File Changes --
> 
>    A DoSpace.jpg (0)
>    A KuL3D.jpg (0)
>    A OmahaMakerGroup.jpg (0)
>    A Tethon3D.png (0)
>    A bootstrap.min.css (6)
>    M index.html (104)
> 
> -- Patch Links --
> 
> https://github.com/saibotsivad/omaha3dprint.com/pull/2.patch
> https://github.com/saibotsivad/omaha3dprint.com/pull/2.diff
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/saibotsivad/omaha3dprint.com/pull/2

## 

Sent from my Android device with K-9 Mail. Please excuse my brevity.

pwnate commented:
Okay! Yeah, I didn't merge it, but I can get to it at some point
On Jan 22, 2016 8:36 PM, "Tobias Davis" notifications@github.com wrote:

> I'm on my phone so I don't know if you merged this already, but I think if
> you merge it it'll get auto deployed? Feel free to merge as much as you
> want!
> 
> On January 22, 2016 7:54:15 PM CST, Nathan Davis notifications@github.com
> wrote:
> 
> > Updated the bootstrap theme. Changed main page layout and added a small
> > amount of content.
> > You can view, comment on, or merge this pull request online at:
> > 
> > https://github.com/saibotsivad/omaha3dprint.com/pull/2
> > 
> > -- Commit Summary --
> > - Started on new website
> > 
> > -- File Changes --
> > 
> > A DoSpace.jpg (0)
> > A KuL3D.jpg (0)
> > A OmahaMakerGroup.jpg (0)
> > A Tethon3D.png (0)
> > A bootstrap.min.css (6)
> > M index.html (104)
> > 
> > -- Patch Links --
> > 
> > https://github.com/saibotsivad/omaha3dprint.com/pull/2.patch
> > https://github.com/saibotsivad/omaha3dprint.com/pull/2.diff
> > 
> > ---
> > 
> > Reply to this email directly or view it on GitHub:
> > https://github.com/saibotsivad/omaha3dprint.com/pull/2
> 
> ## 
> 
> Sent from my Android device with K-9 Mail. Please excuse my brevity.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/saibotsivad/omaha3dprint.com/pull/2#issuecomment-174112482
> .
